core/src/main/java/org/sonarsource/sonarlint/core/websocket/SonarCloudWebSocket.java

```diff
@@ -76,7 +76,7 @@ public void unsubscribe(String projectKey) {
   }
 
   private void send(String messageType, String projectKey) {
-    var unsubscribePayload = new WebSocketEventSubscribePayload(messageType, "QualityGateChanged", projectKey);
+    var unsubscribePayload = new WebSocketEventSubscribePayload(messageType, new String[]{"QualityGateChanged"}, projectKey);
```

Comment:
I notice now that there is a map in this class from event type to parsers. Maybe we could use the map keys instead of duplicating the event type literal?